### PR TITLE
Add Rocs

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** Please ping the KDE team.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/education/rocs/-/merge_requests/37

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.rocs

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by Mailaender** [comment]
bot, build org.kde.rocs

File: org.kde.rocs.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/release-service/22.08.0/src/rocs-22.08.0.tar.xz",
+          "sha256": "b3831934f63fb45910e87a9715b8746ae0072e53a0e0f77162e5dcd6128dac1b",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 8763,
+            "stable-only": true,
+            "url-template": "https://download.kde.org/stable/release-service/$version/rocs-$version.tar.xz"
```
Comment (tsdgeos) [inline org.kde.rocs.json:66]: this is missing a /src/ in the path

**Comment by travier** [comment]
Manifest LGTM, not yet tested. Do we really need full home acces?

**Comment by Mailaender** [comment]
You need to save your files somewhere.

**Comment by hfiguiere** [comment]
> You need to save your files somewhere.

xdg-documents ?

**Comment by travier** [comment]
The file portal should let you save files too. Any issues with it?

**Comment by barthalion** [comment]
bot, build org.kde.rocs

**Pull request closed without merging**